File: python/Level 1/로또의_최고_순위와_최저_순위.py
def solution(lottos, win_nums):
    count = 0
    zero = 0
    for n in lottos:
        if n == 0:
            zero += 1
        else:
            if n in win_nums:
                count += 1

    highRank = min(6, 7 - count - zero)
    lowRank = min(6, 7 - count)

    return [highRank, lowRank]

File: golang/Level 1/로또의_최고_순위와_최저_순위.go
func solution(lottos []int, win_nums []int) []int {
    cnt := 0
    zero := 0
    
    for _, n := range lottos {
        if n == 0 {
            zero++
        } else {
            for _, win_num := range win_nums {
                if n == win_num {
                    cnt++
                    break
                }
            }
        }
    }
    
    highRank := 7 - cnt - zero
    lowRank := 7 - cnt
    if highRank > 6 {
        highRank = 6
    }
    if lowRank > 6 {
        lowRank = 6
    }
    
    return []int{highRank, lowRank}
}